sample-functions/README.md

```diff
@@ -0,0 +1 @@
+Reference functions organized by runtime
```

Comment:
We should put just a bit more into this readme:
* change "Reference" to "Sample"
* put something like...
"Limelight Networks provides these as *examples only*. They are provided *as-is* with no guarantee of correctness or functionality."